src/lib/app/identity.ts

```diff
@@ -72,10 +72,10 @@ export function logout(): void {
 
 /* eslint-disable @typescript-eslint/no-explicit-any */
 /* eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types */
-export async function registerIdentity(username?: string, claimType = UserType.Person, claim?: any): Promise<IdentityJson> {
+export async function registerIdentity(username?: string, claimType = UserType.Person, claim?: any, hidden?: boolean): Promise<IdentityJson> {
```

Comment:
From my testing it seems like the `hidden` parameter is always defined. If thats the case (please confirm on your own) please remove the question mark operator.